wfe/wfe.go

```diff
@@ -1546,40 +1496,34 @@ func (wfe *WebFrontEndImpl) KeyRollover(ctx context.Context, logEvent *requestEv
 	}
 	err = json.Unmarshal(payload, &rolloverRequest)
 	if err != nil {
-		logEvent.AddError("unable to JSON parse resource from JWS payload: %s", err)
```

Comment:
Ditto here: RE losing the unmarshal error

Reply:
I think giving `err` to `sendError` would resolve the concern.

Reply:
See above about not logging detailed JSON parse errors. Also comment below about unintuitive handling of ierr.